### PR TITLE
Throwing the yo

### PR DESCRIPTION
**Pull request by SirWellington**

### CODE REVIEW

File: app/src/main/java/tech/redroma/yoching/extensions/CollectionsPlus.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+
+package tech.redroma.yoching.extensions
+
+import tech.sirwellington.alchemy.arguments.assertions.nonEmptyList
+import tech.sirwellington.alchemy.arguments.checkThat
+
+
+/**
+ *
```
Comment (SirWellington) [inline app/src/main/java/tech/redroma/yoching/extensions/CollectionsPlus.kt]: Hello, add some doc?
bitmoji

File: app/src/main/res/drawable/selectable_button.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+                <shape>
+                    <gradient android:startColor="#BB000000" android:endColor="#7F000000" android:angle="270" />
+                </shape>
+
+            </item>
+
+            <item
+                    android:bottom="3dp"
+            >
```
Comment (SirWellington) [inline app/src/main/res/drawable/selectable_button.xml]: This trailing `>` is weird

File: app/src/main/res/layout/fragment_throw_the_yo.xml [hunk 3]
```diff
@@ -1,125 +1,204 @@
...
+            android:layout_marginRight="8dp"
+            android:layout_marginTop="8dp"
+            android:layout_width="0dp"
+            android:orientation="vertical"
+            app:layout_constraintBottom_toTopOf="@+id/wrexagram_lines_bottom_guideline"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintLeft_toLeftOf="@+id/left"
+            app:layout_constraintRight_toLeftOf="@+id/right"
+            app:layout_constraintTop_toTopOf="@+id/wrexagram_lines_top_guideline"
+    >
```
Comment (SirWellington) [inline app/src/main/res/layout/fragment_throw_the_yo.xml]: Weird

**Comment by SirWellington** [comment]
Deuces
bitmoji

**Comment by Pagalkhan68** [comment]
~~[]()~~

**Pull request merged**